ani-cli

```diff
@@ -628,6 +629,11 @@ if [ -z "$select_first" ]; then
 			ep_choice_start=$((episode + 1))
 			ep_choice_end=""
 			;;
+        	b)
+			prompt "Select quality"
+			quality="$REPLY"
+			video_url=$(get_video_quality "$dpage_link")
+			;;
```

Comment:
Also why not just set `quality`
just this here works
```sh
		b)
			prompt "Select quality"
			quality="$REPLY"
			;;
```I

Reply:
what the point of fetching video_url? we are anyways fetching it open_episode, the current approach does a redundant part which slows the script

Reply:
Okay, will remove that part